### ISSUE TITLE
Rails 5 : SQL with bind parameters when NLS_NUMERIC_CHARACTERS is set to ', ' show Java::JavaSql::SQLSyntaxErrorException: / ORA-01722: invalid number

### ISSUE THREAD

**Issue by yahonda** [rsim/oracle-enhanced#909]
* Steps to reproduce
```
$ cd oracle-enhanced
$ rspec ./spec/active_record/connection_adapters/oracle_enhanced_connection_spec.rb:252

```

* Result

```ruby
jruby-9.1.2.0 [~]$ cd git/oracle-enhanced
jruby-9.1.2.0 [ master ~/git/oracle-enhanced]$ rspec ./spec/active_record/connection_adapters/oracle_enhanced_connection_spec.rb:252
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
uri:classloader:/jruby/kernel/jruby/process_manager.rb:28: warning: executable? does not in this environment and will return a dummy value
==> Loading config from ENV or use default
==> Running specs with JRuby version 9.1.2.0
==> Effective ActiveRecord version 5.0.0
Run options: include {:locations=>{"./spec/active_record/connection_adapters/oracle_enhanced_connection_spec.rb"=>[252]}}
DEPRECATION WARNING: ******************************************************* Passing a column to `bind_param` will be deprecated. `type_casted_binds` should be already type casted so that `bind_param` should not need to know column. ******************************************************* (called from bind_param at /home/yahonda/git/oracle-enhanced/lib/active_record/connection_adapters/oracle_enhanced/jdbc_connection.rb:346)
F

Failures:

  1) OracleEnhancedConnection SQL with bind parameters when NLS_NUMERIC_CHARACTERS is set to ', ' should execute prepared statement with decimal bind parameter
     Failure/Error: Unable to find oracle.jdbc.driver.T4CTTIoer.processError(oracle/jdbc/driver/T4CTTIoer.java to read failed line

     Java::JavaSql::SQLSyntaxErrorException:
       ORA-01722: invalid number
     # oracle.jdbc.driver.T4CTTIoer.processError(oracle/jdbc/driver/T4CTTIoer.java:450)
     # oracle.jdbc.driver.T4CTTIoer.processError(oracle/jdbc/driver/T4CTTIoer.java:399)
     # oracle.jdbc.driver.T4C8Oall.processError(oracle/jdbc/driver/T4C8Oall.java:1059)
     # oracle.jdbc.driver.T4CTTIfun.receive(oracle/jdbc/driver/T4CTTIfun.java:522)
     # oracle.jdbc.driver.T4CTTIfun.doRPC(oracle/jdbc/driver/T4CTTIfun.java:257)
     # oracle.jdbc.driver.T4C8Oall.doOALL(oracle/jdbc/driver/T4C8Oall.java:587)
     # oracle.jdbc.driver.T4CPreparedStatement.doOall8(oracle/jdbc/driver/T4CPreparedStatement.java:225)
     # oracle.jdbc.driver.T4CPreparedStatement.doOall8(oracle/jdbc/driver/T4CPreparedStatement.java:53)
     # oracle.jdbc.driver.T4CPreparedStatement.executeForRows(oracle/jdbc/driver/T4CPreparedStatement.java:943)
     # oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(oracle/jdbc/driver/OracleStatement.java:1150)
     # oracle.jdbc.driver.OraclePreparedStatement.executeInternal(oracle/jdbc/driver/OraclePreparedStatement.java:4798)
     # oracle.jdbc.driver.OraclePreparedStatement.executeQuery(oracle/jdbc/driver/OraclePreparedStatement.java:4845)
     # oracle.jdbc.driver.OraclePreparedStatementWrapper.executeQuery(oracle/jdbc/driver/OraclePreparedStatementWrapper.java:1501)
     # java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:498)
     # org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(org/jruby/javasupport/JavaMethod.java:438)
     # org.jruby.javasupport.JavaMethod.invokeDirect(org/jruby/javasupport/JavaMethod.java:302)
     # RUBY.exec(/home/yahonda/git/oracle-enhanced/lib/active_record/connection_adapters/oracle_enhanced/jdbc_connection.rb:403)
     # RUBY.block in (root)(/home/yahonda/git/oracle-enhanced/spec/active_record/connection_adapters/oracle_enhanced_connection_spec.rb:258)
     # org.jruby.RubyBasicObject.yieldUnder(org/jruby/RubyBasicObject.java:1707)
     # org.jruby.RubyBasicObject.instance_exec(org/jruby/RubyBasicObject.java:1684)
     # RUBY.block in run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:252)
     # RUBY.block in with_around_and_singleton_context_hooks(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:494)
     # RUBY.block in with_around_example_hooks(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:451)
     # RUBY.block in run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/hooks.rb:471)
     # RUBY.run_around_example_hooks_for(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/hooks.rb:609)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/hooks.rb:471)
     # RUBY.with_around_example_hooks(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:451)
     # RUBY.with_around_and_singleton_context_hooks(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:494)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example.rb:249)
     # RUBY.block in run_examples(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example_group.rb:627)
     # org.jruby.RubyArray.collect(org/jruby/RubyArray.java:2341)
     # org.jruby.RubyArray.map(org/jruby/RubyArray.java:2355)
     # RUBY.run_examples(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example_group.rb:623)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example_group.rb:589)
     # RUBY.block in run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example_group.rb:590)
     # org.jruby.RubyArray.collect(org/jruby/RubyArray.java:2341)
     # org.jruby.RubyArray.map(org/jruby/RubyArray.java:2355)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/example_group.rb:590)
     # RUBY.block in run_specs(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:113)
     # org.jruby.RubyArray.collect(org/jruby/RubyArray.java:2341)
     # org.jruby.RubyArray.map(org/jruby/RubyArray.java:2355)
     # RUBY.block in run_specs(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:113)
     # RUBY.with_suite_hooks(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/configuration.rb:1836)
     # RUBY.block in run_specs(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:112)
     # RUBY.report(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/reporter.rb:77)
     # RUBY.run_specs(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:111)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:87)
     # RUBY.run(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:71)
     # RUBY.invoke(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/lib/rspec/core/runner.rb:45)
     # RUBY.<top>(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/rspec-core-3.5.1/exe/rspec:4)
     # org.jruby.Ruby.runInterpreter(org/jruby/Ruby.java:849)
     # org.jruby.Ruby.loadFile(org/jruby/Ruby.java:2986)
     # org.jruby.RubyKernel.loadCommon(org/jruby/RubyKernel.java:970)
     # org.jruby.RubyKernel.load(org/jruby/RubyKernel.java:962)
     # RUBY.<top>(/home/yahonda/.rbenv/versions/jruby-9.1.2.0/bin/rspec:1)
     # org.jruby.Ruby.runInterpreter(org/jruby/Ruby.java:868)
     # org.jruby.Ruby.runInterpreter(org/jruby/Ruby.java:873)
     # org.jruby.Ruby.runNormally(org/jruby/Ruby.java:765)
     # org.jruby.Ruby.runFromMain(org/jruby/Ruby.java:579)
     # org.jruby.Main.doRunFromMain(org/jruby/Main.java:425)
     # org.jruby.Main.internalRun(org/jruby/Main.java:313)
     # org.jruby.Main.run(org/jruby/Main.java:242)
     # org.jruby.Main.main(org/jruby/Main.java:204)

Finished in 0.966 seconds (files took 6.22 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/active_record/connection_adapters/oracle_enhanced_connection_spec.rb:252 # OracleEnhancedConnection SQL with bind parameters when NLS_NUMERIC_CHARACTERS is set to ', ' should execute prepared statement with decimal bind parameter

$
``` 

* Environment
```
Rails 5 `5-0-stable` branch
Oracle enhanced adapter master branch
jruby 9.1.2.0 (2.3.0) 2016-05-26 7357c8f Java HotSpot(TM) 64-Bit Server VM 25.92-b14 on 1.8.0_92-b14 +jit [linux-x86_64]
java version "1.8.0_92"
Oracle Database 12c 12.1.0.2
```

**Comment by yahonda** [comment]
Related with #904

**Comment by yahonda** [comment]
Closed as #929 has been merged to master.
